prov/efa/configure.m4

```diff
@@ -134,6 +134,15 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 			[efadv_support_extended_cq=0])
 	      ])
 
+	AS_IF([test $efadv_support_extended_cq -eq 1],
+		[AC_EGREP_HEADER(
```

Comment:
Thank you!

Maybe add a comment here noting that the reason we want to to use extended CQ is because it provided extra functionality like `read_sgid`.  Therefore if rdma-core does not support read sgid, we do not want to use extended CQ.